sources/constants.js

```diff
@@ -135,6 +135,7 @@ export default {
 	PATH_DOWNLOAD_PDF_TERMS_OF_USE: "/sources/filesForDownload/termsOfUse.pdf",
 	PATH_DOWNLOAD_TXT_TERMS_OF_USE: "/sources/filesForDownload/termsOfUse.txt",
 
+	URL_ISIC_LANDING_PAGE: "https://isic-archive.com/",
```

Comment:
```suggestion
	URL_ISIC_LANDING_PAGE: "https://www.isic-archive.com/",
```

The new landing page is at `www.`